oauth2_http/javatests/com/google/auth/oauth2/DownscopedCredentialsTest.java

```diff
@@ -224,7 +224,7 @@ private static GoogleCredentials getServiceAccountSourceCredentials(boolean canR
       transportFactory.transport.setError(new IOException());
     }
 
-    return sourceCredentials;
+    return sourceCredentials.createScoped("https://www.googleapis.com/auth/cloud-platform");
```

Comment:
Is this a default?

Reply:
Yes, because we're using ServiceAccountCredentials.